src/physics/em/detail/EPlusGGInteractor.hh

```diff
@@ -9,17 +9,39 @@
 
 #include "base/Macros.hh"
 #include "base/Types.hh"
+#include "base/StackAllocator.hh"
 #include "physics/base/Interaction.hh"
 #include "physics/base/ParticleTrackView.hh"
-#include "base/StackAllocator.hh"
 #include "physics/base/Secondary.hh"
 #include "physics/base/Units.hh"
-#include "EPlusGG.hh"
 
 namespace celeritas
 {
 namespace detail
 {
+//---------------------------------------------------------------------------//
+/*!
+ * Device data for creating an EPlusGGInteractor.
+ */
+struct EPlusGGPointers
```

Comment:
As a to-do we should normalize the nomenclature for pointers/data/ref/containergroup.

Reply:
Agreed, I also noticed some inconsistent nomenclature as I was working through this